Traceback (most recent call last):
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/vdsukhov/mambaforge/envs/jb/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
nissan_gtr = Car(brand="Tesla", model="X", year=2023, color="black", horsepower=565)
cars = [tesla_x, camry, nissan_gtr]
print(sorted(cars))
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[15], line 3[0m
[1;32m      1[0m nissan_gtr [38;5;241m=[39m Car(brand[38;5;241m=[39m[38;5;124m"[39m[38;5;124mTesla[39m[38;5;124m"[39m, model[38;5;241m=[39m[38;5;124m"[39m[38;5;124mX[39m[38;5;124m"[39m, year[38;5;241m=[39m[38;5;241m2023[39m, color[38;5;241m=[39m[38;5;124m"[39m[38;5;124mblack[39m[38;5;124m"[39m, horsepower[38;5;241m=[39m[38;5;241m565[39m)
[1;32m      2[0m cars [38;5;241m=[39m [tesla_x, camry, nissan_gtr]
[0;32m----> 3[0m [38;5;28mprint[39m([38;5;28;43msorted[39;49m[43m([49m[43mcars[49m[43m)[49m)

[0;31mTypeError[0m: '<' not supported between instances of 'Car' and 'Car'

